tools/cli/go-whisk/whisk/action.go

```diff
@@ -122,33 +122,39 @@ func (action Action) WebAction() (webExportValue bool) {
 Returns the URL of an action as a string. A valid API host, path and version must be passed. A package that contains the
 action must be passed as well. An empty string must be passed if the action is not packaged.
  */
-func (action Action) ActionURL(apiHost string, apiPath string, apiVersion string, pkg string) (actionURL string) {
-    webActionPath := "https://%s%s/%s/web/%s/%s/%s"
-    actionPath := "https://%s%s/%s/namespaces/%s/actions/%s"
+func (action Action) ActionURL(apiHost string, apiPath string, apiVersion string, pkg string) (string, error) {
+    baseURL, err := GetURLBase(apiHost, apiPath)
+    if err != nil {
+        Debug(DbgError, "getURLBase(%s, %s) failed: %s\n", apiHost, apiPath, err)
```

Comment:
Wrong debug message here. `getURLBase` should be `GetURLBase`.

Reply:
I was unsure if this was meant to be the exact method name in the debug message or if it just happened to match the old name. I'll update these.